relayer/src/worker/client.rs

```diff
@@ -49,7 +53,8 @@ impl<ChainA: ChainHandle, ChainB: ChainHandle> ClientWorker<ChainA, ChainB> {
         );
 
         // initial check for evidence of misbehaviour for all updates
-        let skip_misbehaviour = self.detect_misbehaviour(&client, None);
+        let skip_misbehaviour =
```

Comment:
Just above is this log:
```
        info!(
            "[{}] running client worker & initial misbehaviour detection",
            client
        );
```
can we drop "initial misbehavior detection" part if misbehavior is not configured? log "refresh" and "misbehavior" depending on the config.

Reply:
https://github.com/informalsystems/ibc-rs/pull/1539/commits/db3b6a5a11eff6f85f7f5f3ee64d4e87bc092a87